src/perform_step/low_order_rk_perform_step.jl

```diff
@@ -617,10 +629,16 @@ end
   k3 = f(uprev+dt*(a31*k1+a32*k2), p, t+c2*dt)
   k4 = f(uprev+dt*(a41*k1+a42*k2+a43*k3), p, t+c3*dt)
   k5 = f(uprev+dt*(a51*k1+a52*k2+a53*k3+a54*k4), p, t+c4*dt)
-  k6 = f(uprev+dt*(a61*k1+a62*k2+a63*k3+a64*k4+a65*k5), p, t+dt)
+  g6 = uprev+dt*(a61*k1+a62*k2+a63*k3+a64*k4+a65*k5)
+  k6 = f(g6, p, t+dt)
   update = a71*k1+a73*k3+a74*k4+a75*k5+a76*k6
   u = uprev+dt*update
   integrator.fsallast = f(u, p, t+dt); k7 = integrator.fsallast
+  g7 = u
+  # Hairer II, page 22
+  ϱ = norm(k7 - k6, Inf)/norm(g7 - g6, Inf)
```

Comment:
Reason for the Inf norm? Also, is there a way to make this non-allocating (that's safe given the way the aliases in the cache are)